src/components/input/bl-input.css

```diff
@@ -0,0 +1,159 @@
+:host {
+  display: inline-block;
+  width: 200px;
+  position: relative;
+
+  --bl-input-padding-vertical: var(--bl-size-2xs);
+  --bl-input-padding-horizontal: var(--bl-size-xs);
+  --bl-input-border-color: var(--bl-color-border);
+  --bl-input-icon-color: var(--bl-color-content-tertiary);
+  --bl-input-text-color: var(--bl-color-content-primary);
+}
+
+input {
+  outline: none;
+  box-sizing: border-box;
+  border: none;
+  width: 100%;
+  font: var(--bl-font-title-3-regular);
+  padding: var(--bl-input-padding-vertical) var(--bl-input-padding-horizontal);
+  margin: 0;
+  box-shadow: inset 0 0 0 1px var(--bl-input-border-color);
```

Comment:
is box-shadow necessary?

Reply:
Not anymore, I just changed them to borders. But as I mentioned in [my previous comment](https://github.com/Trendyol/baklava/pull/129#issuecomment-1171220747) we just changed the approach about setting the height of the components.